pages/interview-process.md

```diff
@@ -4,7 +4,7 @@ title: Interview process
 ---
 
 ## Application collection
-Applications are collected and grouped in two-week intervals. All applications collected within each two-week period are treated as a cohort and evaluated against each other.
+Applications are collected during the period specified by the job posting, usually 5 days to two weeks. All applications collected within this period are treated as a cohort and evaluated against each other.
```

Comment:
Please change `5` to `five` for consistency. After that, this is good to go, IMO.